src/api/privateApi.js

```diff
@@ -61,111 +61,92 @@ export function getURLPrefix() {
 
 /**
  * @method getFullURL
- * @return url string
+ * @return {String}
  * @private
  */
 
 export function getFullURL() {
 	const nodeUrl = this.port
-		? `${this.currentPeer}:${this.port}`
-		: this.currentPeer;
+		? `${this.node}:${this.port}`
+		: this.node;
 
 	return `${getURLPrefix.call(this)}://${nodeUrl}`;
 }
 
 /**
- * @method getPeers
- * @return peers Array
+ * @method getNodes
+ * @return {Array}
  * @private
  */
 
-export function getPeers() {
-	if (this.testnet) return this.defaultTestnetPeers;
-	if (this.ssl) return this.defaultSSLPeers;
-	return this.defaultPeers;
+export function getNodes() {
+	if (this.testnet) return this.defaultTestnetNodes;
+	if (this.ssl) return this.defaultSSLNodes;
+	return this.defaultNodes;
 }
 
 /**
- * @method getRandomPeer
- * @return peer string
+ * @method getRandomNode
+ * @return  {String}
  * @private
  */
 
-export function getRandomPeer() {
-	const peers = getPeers.call(this)
-		.filter(peer => !this.bannedPeers.includes(peer));
+export function getRandomNode() {
+	const nodes = getNodes.call(this)
+		.filter(node => !this.bannedNodes.includes(node));
```

Comment:
We check against `this.bannedNodes.includes(node)` a lot. A new `privateApi` method `isBanned` would make these checks more DRY.